recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend

```diff
@@ -8,6 +8,13 @@ XT_GUESTS_BUILD ?= "doma"
 XT_GUESTS_INSTALL ?= "doma"
 
 python __anonymous () {
+    machine = d.getVar('MACHINE', True)
+    not_supported_machines = ["salvator-x-m3", "salvator-xs-h3", "h3ulcb", "m3ulcb", "salvator-x-h3"]
+    if machine in not_supported_machines:
+        warning = 'The machine {} is not supported.'.format(machine) 
+        bb.warn(warning)
+        bb.fatal("Use branch prod-process-p2m-lookup of prod-devel to suupport.")
```

Comment:
branch "prod-process-p2m-lookup" should be renamed

Reply:
fixed at
if machine in not_supported_machines:
        warning = 'The machine {} is not supported.'.format(machine) 
        bb.warn(warning)
        bb.fatal("Use prod-devel-unsafe of prod-devel to support.")